src/components/PoolOverview.tsx

```diff
@@ -44,56 +44,76 @@ function PoolOverview({
     }),
   }
 
+  const hasShare = parseFloat(formattedData.userBalanceUSD) ? true : false
```

Comment:
I wouldn't expect parseFloat to work with commified values, plus we prefer not to parse formatted values when we can just use the raw value
`const hasShare = !!userShareData?.usdBalance.gt("0")`